nemo_rl/algorithms/loss_functions.py

```
@@ -168,6 +168,50 @@ def __call__(
+
+        # policy-kl(kl(P_train || P_gen)) = torch.exp(log_ratio) - log_ratio - 1
+        # where log_ratio = prev_logprobs - generation_logprobs
+        policy_kl = masked_mean(
+            torch.exp(generation_logprobs - prev_logprobs)
+            - (generation_logprobs - prev_logprobs)
+            - 1,
+            mask,
+            global_normalization_factor=global_valid_toks,
+        ).item()
+
+        # Jensen–Shannon divergence
```

Ruff (0.14.1)

Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)